paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -210,30 +212,6 @@ internal class PaymentSheetActivityTest {
         }
     }
 
-    @Test
-    fun `google pay button should not be enabled when editing`() {
```

Comment:
This is already tested in the view model tests (https://github.com/stripe/stripe-android/blob/3c9768252d59a56a771cc355e6c98f1da3818944/paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt#L688).